pages.de/common/cat.md

```diff
@@ -9,16 +9,16 @@
 
 - Verkette mehrere Dateien und speichere das Ergebnis in einer neuen Datei:
 
-`cat {{pfad/zu/datei1}} {{pfad/zu/datei2}} > {{pfad/zu/ziel_datei}}`
+`cat {{pfad/zu/datei1 pfad/zu/datei2 ...}} > {{pfad/zu/ziel_datei}}`
 
 - Verkette mehrere Dateien und hänge das Ergebnis an eine Datei an:
 
-`cat {{pfad/zu/datei1}} {{pfad/zu/datei2}} >> {{pfad/zu/ziel_datei}}`
+`cat {{pfad/zu/datei1 pfad/zu/datei2 ...}} >> {{pfad/zu/ziel_datei}}`
 
-- Nummeriere alle ausgegebenen Zeilen:
+- Kopieren den Inhalt einer Datei ohne Pufferung in eine Ausgabedatei:
 
-`cat -n {{pfad/zu/datei}}`
+`cat -u {{/dev/tty12}} > {{/dev/tty13}}`
```

Comment:
For me it says that -u is ignored: https://manned.org/cat.1

Reply:
This flag is indeed ignored in some versions, but I wouldn't suggest removing it in `common` as it is still supported in some places i.e. https://manned.org/man/cat.1posix. Instead, we can remove it on the `linux` variant of the page alone. (Regarding `-n` I don't think we should add it in the `common` page, see the last section of above manpage for more information)